src/code42cli/cmds/users.py

```diff
@@ -122,3 +135,37 @@ def _get_users_dataframe(sdk, columns, org_uid, role_id, active):
         users_list.extend(page["users"])
 
     return DataFrame.from_records(users_list, columns=columns)
+
+
+def _add_legal_hold_membership_to_user_dataframe(sdk, df):
+    columns = ["legalHold.legalHoldUid", "legalHold.name", "user.userUid"]
+
+    legal_hold_member_dataframe = (
+        json_normalize(list(_get_all_active_hold_memberships(sdk)))[columns]
+        .groupby(["user.userUid"])
+        .agg(",".join)
+        .rename(
+            {
+                "legalHold.legalHoldUid": "legalHoldUid",
```

Comment:
In the command for `code42 legal-hold list`, for table format, we should this as `Matter ID`... Will that also happen here?

Reply:
It would be nice to be able to pass in user friendly keys for Table format headers for when using `DataFrameOutputFormatter`, as that was the norm before.

Table format should use label-esque keys while all the other ones should use raw API values.